docs/KNOWLEDGE_BASE.md

```
@@ -29,3 +29,22 @@ This file contains "Evergreen" notes—verified facts, patterns, and solutions t
     - **Context/State Management:** Agents missing inputs.
     - **Path/File Handling:** Relative path issues.
     - **Tool Invocation:** Incorrect arguments to subprocesses.
+
+### Wide Events (ADR 015)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)